lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -2523,6 +2524,28 @@ bool SwiftLanguageRuntimeImpl::GetDynamicTypeAndAddress_IndirectEnumCase(
   }
 }
 
+void SwiftLanguageRuntimeImpl::DumpTyperef(
+    CompilerType type, TypeSystemSwiftTypeRef *module_holder,
+    SwiftASTContext *swift_ast_context, Stream *s) {
+  if (!s)
+    return;
+  const auto *typeref = GetTypeRef(type, module_holder, swift_ast_context);
+  if (!typeref)
+    return;
+  // Hacky: since dump takes a FILE *, create a temporary file, read into it,
+  // then read its contents...
+  std::FILE *temp_file = std::tmpfile();
```

Comment:
we have an llvm function to get temporary files, too (no idea if it's easy to turn them into a FILE*).